torchvision/prototype/models/vision_transformer.py

```diff
@@ -441,78 +225,3 @@ def vit_l_32(*, weights: Optional[ViT_L_32_Weights] = None, progress: bool = Tru
         progress=progress,
         **kwargs,
     )
-
```

Comment:
Github won't let me place this comment on the right place, but could you please remove the documentation docstrings from the methods on prototype? This avoids having duplicate documentation to maintain. Note that the docstrings are not parsed in the documentation so they have no effect.